server/public/css/events.css

```diff
@@ -28,183 +7,132 @@
   overflow-y: auto;
 }
 
-#events .sequence-section > .sequence-event {
+.sequence__event {
   display: flex;
   flex-direction: column;
   align-items: center;
 }
 
-#events .sequence-section > .sequence-event ~ .sequence-event {
+.sequence__event ~ .sequence__event {
   margin-top: 5px;
 }
 
-#events .sequence-section > .sequence-event > div:not(:nth-child(2)) {
+.sequence__event > div:not(:nth-child(2)) {
   width: 100%;
   height: 5px;
   background-color: transparent;
 }
 
-#events .sequence-tag {
-  align-items: center;
-  display: flex;
-  background-color: rgb(178 229 178);
-  border: 2px solid green;
-  border-radius: 5px;
-  cursor: grab;
-  padding: 5px;
-  min-width: 100px;
-  margin: 0;
-  justify-content: space-between;
-}
-
-#events .sequence-tag svg {
-  cursor: pointer;
-  height: 20px;
-  width: 20px;
-}
-
-#events .sequence-tag .event-actions {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    margin-left: 10px;
-}
-
-#events .sequence-tag_ghost-image {
-  cursor: grabbing;
-  font-size: 0.5rem;
-}
-
-#events .sequence_actions {
+.sequence__actions {
   display: flex;
   justify-content: flex-end;
 }
 
-#events .sequence_actions > button {
+.sequence__action {
   background-color: transparent;
   border: unset;
   cursor: pointer;
 }
 
-#events .sequence_actions > button > svg {
+.sequence__icon {
   height: 20px;
   width: 20px;
 }
 
-/* left column */
-#events .events-section {
-  display: flex;
-  flex-direction: column;
-  height: 100%;
-}
-
-#events .events-section .events_wrapper {
-  height: 200px;
-  overflow-y: auto;
-}
 
-#events .events-section .events-type-wrapper {
+/* EVENTS */
+.events {
   display: flex;
-  margin-bottom: 10px;
-  height: 30px;
-  align-items: center;
-}
-
-#events .events-section .events-type-wrapper div {
-  display: flex;
-  align-items: center;
-  border-right: 1px solid;
-  cursor: pointer;
+  flex-direction: column;
   height: 100%;
-  min-width: 100px;
-  padding: 10px 5px;
-  margin: 0;
 }
 
-#events .events-list {
+.events__list {
   align-content: flex-start;
-  display: grid;
+  display: none;
   grid-template-columns: 1fr 1fr;
-  gap: 20px;
+  gap: 1rem;
   overflow: scroll;
   height: 62%;
 }
 
-#events .sequence-tag p {
-  margin: 0;
-  width: 100%;
+.events__list.is--active {
+  display: grid;
 }
 
+/* EVENTS - CUSTOM */
+.events__custom {
+  display: none;
+}
+
+.events__custom.is--active {
+  display: block;
+}
 
-#events .form {
+.events__form {
   display: flex;
   flex-direction: column;
   align-items: flex-start;
 }
 
-#events .dragged_over {
+.dragged_over {
   transform: translateX(100%);
 }
 
-#events .new_sequence {
+.new_sequence {
   border: unset;
   font-size: 1.2rem;
   font-weight: bold;
   width: 100%;
 }
 
-#events .new_sequence:focus-visible,
-#events .new_sequence:focus {
+.new_sequence:focus-visible,
+.new_sequence:focus {
   border: unset;
   outline: unset;
 }
 
-#events .form .form_block {
+.events__form_block {
```

Comment:
Following BEM rulese, shouldn't this be `.events__form-block`?

We did like this for `.events__form-column`

Reply:
yup, missed it